test/functional/services/dashboard/visualizations.js

```diff
@@ -81,6 +108,8 @@ export function DashboardVisualizationProvider({ getService, getPageObjects }) {
       if (checkForAddPanel) {
         await dashboardAddPanel.ensureAddPanelIsShowing();
         await dashboardAddPanel.clickAddNewEmbeddableLink('visualization');
```

Comment:
I could find no case where `checkForAddPanel` has a other value then `false`, so I think this can be cleaned up?

Reply:
Good point, I wasn't sure if it's the case, so I left it.